frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -107,8 +107,7 @@ def process_spider_output(self, response, result, spider):
         for element in result:
             if isinstance(element, Request):
                 links.append(element)
-            else:
-                yield element
+            yield element
```

Comment:
Hey @amleczko! Thanks for the PR! Unfortunately, it doesn't look like it solves the problem. IMO, the right approach would be to adopt the whole Frontera pipeline for method ``page_crawled`` in backends and manager to support generators. Can you afford this? I'm ready to guide you through if you're up for this.